src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1242,6 +1283,7 @@ private void onDecodePublish(
 
                 final MqttDataExFW dataEx = builder.build();
                 stream.doApplicationData(traceId, authorization, reserved, payload, dataEx);
+                doSignalKeepAliveTimeoutIfNecessary();
```

Comment:
Just need to keep track of latest packet arrival time, for use in `onKeepAliveTimeoutSignal` to extend the deadline, same for other inbound network frames.

Initial signal will already be scheduled in `onDecodeConnect` and subsequent signals will be scheduled in `onKeepAliveTimeoutSignal` so don't need to trigger scheduling from here, right?